packages/mui-material-next/tsconfig.build.json

```diff
@@ -13,8 +13,8 @@
   "include": ["./src/**/*.ts*"],
   "exclude": ["src/**/*.spec.ts*", "src/**/*.test.ts*"],
   "references": [
+    { "path": "../mui-utils/tsconfig.build.json" },
```

Comment:
I had to fix the `references` in all the packages, otherwise TS was not able to resolve correctly the imports.
Not sure why it is impacted by the new structure